commit d6e36ec1d04edec73fedeea89ec8027e3d215dcb
Author: Helmut Schaa <helmut.schaa@googlemail.com>
Date:   Mon Mar 15 17:22:26 2010 +0100

    rt2x00: fix warning when building rt2800pci with just soc support
    
    Fix compile warning "rt2800pci.c:1248: warning: 'rt2800pci_device_table'
    defined but not used" when building rt2800pci with only soc support
    (without pci).
    
    Signed-off-by: Helmut Schaa <helmut.schaa@googlemail.com>
    Acked-by: Ivo van Doorn <IvDoorn@gmail.com>
    Signed-off-by: John W. Linville <linville@tuxdriver.com>

--- a/drivers/net/wireless/rt2x00/rt2800pci.c
+++ b/drivers/net/wireless/rt2x00/rt2800pci.c
@@ -1184,6 +1184,7 @@ static const struct rt2x00_ops rt2800pci
 /*
  * RT2800pci module information.
  */
+#ifdef CONFIG_RT2800PCI_PCI
 static DEFINE_PCI_DEVICE_TABLE(rt2800pci_device_table) = {
 	{ PCI_DEVICE(0x1814, 0x0601), PCI_DEVICE_DATA(&rt2800pci_ops) },
 	{ PCI_DEVICE(0x1814, 0x0681), PCI_DEVICE_DATA(&rt2800pci_ops) },
@@ -1211,6 +1212,7 @@ static DEFINE_PCI_DEVICE_TABLE(rt2800pci
 #endif
 	{ 0, }
 };
+#endif /* CONFIG_RT2800PCI_PCI */
 
 MODULE_AUTHOR(DRV_PROJECT);
 MODULE_VERSION(DRV_VERSION);
